tests/eas/generic.py

```diff
@@ -306,7 +306,6 @@ def plot_expected_util(self, experiment, util_df):
                 color = 'red'
 
             tdf.plot(ax=ax[cpu], drawstyle='steps-post', title="CPU{}".format(cpu), color=color)
-            ax[cpu].fill_between(tdf.index, tdf, 0, step='post', color=color)
```

Comment:
Does it make sense to test and use the method if the backend supports it?
I guess this allows to get better pretty locking plots when we run tests from out desktop....

Reply:
I had a quick look for that and it didn't seem very intuitive - that "step" parameter is part of the `fill_between` API so imho it should be supported by all of them. I have a post-it somewhere to remind me to look at that again sometime in the future...

Reply:
What about just a simple try: except pass?

Reply:
Not too hot on that but we could do it. 

> I guess this allows to get better pretty locking plots when we run tests from out desktop....

For that you'll also have to change the backend definied in tests/__init__.py

Reply:
As per chat we had, seems that fixing this is not enough to allow the removal of the `Agg` backend.
So, let's got with this version.